packages/cli-e2e/utils/browser.ts

```diff
@@ -50,6 +54,21 @@ export async function getNewBrowser(): Promise<Browser> {
   });
 }
 
+export async function captureScreenshots(browser: Browser): Promise<void> {
+  mkdirSync(SCREENSHOTS_PATH, {recursive: true});
+  for (const page of await browser.pages()) {
+    try {
+      await page.screenshot({
+        fullPage: true,
+        type: 'png',
+        path: resolve(SCREENSHOTS_PATH, expect.getState.name),
```

Comment:
Should we clean the string before (remove dots for instance)? Maybe it's not important.